packages/core/src/env-set/index.ts

```diff
@@ -62,7 +62,7 @@ function createEnvSet() {
 
     load: async () => {
       values = await loadEnvValues();
-      pool = await createPoolByEnv(values.isTest);
+      pool = await createPoolByEnv(values.isTest, `${values.localhostUrl}/${MountedApps.DemoApp}`);
```

Comment:
Hmmm, why do we have to dynamically load demo-app path? Can we just hard-code it?

Reply:
go through the code you will find why

Reply:
go through the code you will find why

Reply:
basically it's not "dynamically load", just for seed

Reply:
OK, I did, and I think what we actually need here is the `values.localhostUrl`. Then how about we change the param name to `localhostUrl`, because create DB pool with a `demoAppUrl` looks a little bit strange.

Reply:
OK, we can't. I get it now.